openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 
-<div class="staff__grade__control ui-toggle-visibility is--collapsed">
-    <header class="staff__grade__header ui-toggle-visibility__control">
+<div class="staff__grade__control ui-slidable">
+    <header class="staff__grade__header ui-slidable__control">
         <h3 class="staff__grade__title">
             <span class="wrapper--copy">
                 <button class="staff__grade__show-form">{% trans "Staff Assessment" %}</button>
```

Comment:
@cptvitamin note that I chose (in a previous PR) to make this control a button so that keyboard focus is handled correctly.